wled00/data/settings_time.htm

```diff
@@ -190,6 +190,7 @@ <h3>Clock</h3>
 			12h LED: <input name="OM" type="number" min="0" max="255" required><br>
 			Show 5min marks: <input type="checkbox" name="O5"><br>
 			Seconds (as trail): <input type="checkbox" name="OS"><br>
+			Add pointer colors: <input type="checkbox" name="OA"><br>
```

Comment:
Even though technically it is adding colors, the better term would be "mix".

Reply:
I need the feature just so it is visible that minute and second (and possibly hour) are shown at the same pixel. Usually the segment is solid black before the overlay. But mixing the pointers into the effect's colors opens up a range of new possibilities too :-)

I found the function AddPreserveRatio in the PR, looks promising. 

Maybe after it's merged I'll make the option a dropdown to choose from a bunch of merge options?

Reply:
That is good enough for me. 👍 
Even though that particular PR may not get merged I will ask the author to make a new one featuring just additive function.